src/components/Layout/Header/GlobalSearch/GlobalSearchButton.tsx

```diff
@@ -62,10 +75,12 @@ export const GlobalSeachButton = () => {
   const lpOpportunities = useAppSelector(selectAggregatedEarnUserLpOpportunities)
   const globalSearchFilter = useMemo(() => ({ searchQuery }), [searchQuery])
   const results = useAppSelector(state => selectGlobalItemsFromFilter(state, globalSearchFilter))
-  const [assetResults, stakingResults, lpResults, txResults] = results
-  const flatResults = useMemo(() => results.flat(), [results])
+  const allResults = (results as any).concat(sendResults) as ResultsWithActions
+  const [assetResults, stakingResults, lpResults, txResults] = allResults
+  const flatResults = useMemo(() => allResults.flat(), [allResults])
```

Comment:
could just do this and then we don't need `ResultsWithActions` type or the `any` `as` casting
```suggestion
  const [assetResults, stakingResults, lpResults, txResults] = results
  const flatResults = useMemo(() => [...results, sendResults].flat(), [results, sendResults])
```

Reply:
618fe5dd7